yass-components-react/src/components/Box/stories.tsx

```diff
@@ -0,0 +1,196 @@
+      options: classNames["padding-block"],
+    },
+    paddingInline: {
+      control: { type: "select" },
+      options: classNames["padding-inline"],
+    },
+    backgroundColor: {
+      control: { type: "select" },
+      options: classNames["background-color"],
+    },
+    borderWidth: {
+      control: { type: "select" },
+      options: classNames["border-width"],
+    },
+    borderColor: {
+      control: { type: "select" },
+      options: classNames["border-color"],
+    },
+    borderStyle: {
+      control: { type: "select" },
+      options: classNames["border-style"],
+    },
+    borderRadius: {
+      control: { type: "select" },
+      options: classNames["border-radius"],
+    },
+    overflow: {
+      control: { type: "select" },
+      options: classNames["overflow"],
+    },
+    opacity: {
+      control: { type: "select" },
+      options: classNames["opacity"],
+    },
+    zIndex: {
+      control: { type: "select" },
+      options: classNames["z-index"],
+    },
+    textColor: {
+      control: { type: "select" },
+      options: classNames["color"],
+    },
+  },
+} as ComponentMeta<typeof Box>;
+
+const DefaultTemplate: ComponentStory<typeof Box> = ({
+  children,
+  paddingBlock,
+  paddingInline,
+  borderColor,
+  backgroundColor,
+  borderStyle,
+  borderWidth,
+  borderRadius,
+  overflow,
+  opacity,
+  zIndex,
+  // @ts-ignore TODO: Better story types
+  textColor,
+}) => (
+  <Box
+    paddingBlock={paddingBlock}
+    paddingInline={paddingInline}
+    borderColor={borderColor}
+    backgroundColor={backgroundColor}
+    borderStyle={borderStyle}
+    borderWidth={borderWidth}
+    borderRadius={borderRadius}
+    overflow={overflow}
+    opacity={opacity}
+    zIndex={zIndex}
+  >
+    <Text color={textColor}>{children}</Text>
+  </Box>
+);
+
+export const Default = DefaultTemplate.bind({});
+Default.args = {
+  children: "Some content in a box",
+  paddingBlock: "padding:10",
+  paddingInline: "padding:5",
+  borderColor: "border-color:blue-400",
+  backgroundColor: "background-color:blue-200",
+  borderStyle: "border-style:solid",
+  borderWidth: "border-width:2",
+  borderRadius: "border-radius:3",
+  overflow: "overflow:initial",
+  opacity: "opacity:1000",
+  zIndex: "z-index:100",
+};
+
+const colors = [
+  "neutral",
+  "red",
+  "orange",
+  "yellow",
+  "green",
+  "teal",
+  "blue",
+  "purple",
+  "pink",
+];
+const values = [
+  "50",
+  "100",
+  "200",
+  "300",
+  "400",
+  "500",
+  "600",
+  "700",
+  "800",
+  "900",
+];
+
+export const Colors: ComponentStory<typeof Box> = () => (
+  <Stack gap="gap:2">
+    {colors.map((color: string) => (
+      <Inline gap="gap:2">
+        {values.map((value: string) => (
+          <Box
+            padding="padding:20"
+            backgroundColor={`background-color:${color}-${value}`}
+          ></Box>
+        ))}
+      </Inline>
+    ))}
+  </Stack>
+);
+
+export const Opacity: ComponentStory<typeof Box> = () => (
```

Comment:
Oh, I originally had `opacity` range from 0 to 100, so that it corresponded to a percentage, but I changed it last minute, because it felt odd that it wasn't a milennial scale like everything else. Unfortunate side-effect is that `opacity: 100%` is `opacity:1000`. Lemme know if you have a strong opinion
 
<img width="1250" alt="image" src="https://user-images.githubusercontent.com/4335944/205898262-5a9b715b-77e5-4106-a438-61147b9f2c8c.png">